_includes/_global_js.html

```diff
@@ -0,0 +1,55 @@
+<!-- This should be inserted in the page *before* any page-specific JS -->
+<script src="/assets/js/jquery.js"></script>
+<script src="/assets/js/bootstrap.min.js"></script>
```

Comment:
Duplicate includes, both bootstrap and jquery are already included in the `base.html` layout.